api_EPA.py

```diff
@@ -3,17 +3,19 @@
 import os
 import requests
 import json
-import time
+import datetime
+from time import gmtime, strftime
 from io import StringIO
+from redivis import bigquery
 
 # When calling this script, provide the API Token as an environment variables
 # E.g.: REDIVIS_API_TOKEN=your_api_token pipenv run python upload_script.py
 
 # This script assumes you have an API access token with date.edit scope. See https://apidocs.redivis.com/authorization
-access_token = os.environ["REDIVIS_API_TOKEN"]
+access_token = "AAAAYq9097o3xhmyEpN5iNqtZGPJMA7a"
```

Comment:
As a rule of thumb it's a very bad idea to commit access tokens (these effectively act as passwords and allow anyone w/ the token to act on your behalf). 

We should continue with the strategy of using the `REDIVIS_API_TOKEN` as an environment variable - e.g.:
```sh
REDIVIS_API_TOKEN=AAA... pipenv run python api_EPA.py
```
This will also allow other users to run this script with their own token.

Finally, it would be good to delete this token on Redivis and issue a new one, since this is a public repo and anyone can have access to its git history